lib/kubernetes-deploy/kubernetes_resource.rb

```diff
@@ -201,7 +206,8 @@ def self.go_template_for(kind, name)
           %[(eq .involvedObject.kind "#{kind}")],
           %[(eq .involvedObject.name "#{name}")],
           '(ne .reason "Started")',
-          '(ne .reason "Created")'
+          '(ne .reason "Created")',
+          '(ne .reason "SuccessfulCreate")'
```

Comment:
Personally I like seeing those, but maybe you're right that we should reduce noise. I won't object to filtering all 4.